### ISSUE TITLE
Run CI on CentOS 7

### ISSUE THREAD

**Issue by andrewsykim** [moby/ipvs#20]
A lot of Kubernetes users (using IPVS proxy) run on CentOS 7 which is based on Linux 3.10. In the past there have been compatibility issues with older kernel where certain netlink attributes do not exist. For example, for Linux < 3.18 the address family doesn't exist so we had to create some workarounds (https://github.com/moby/ipvs/pull/15 and https://github.com/moby/ipvs/pull/19). The address family compatibility issue would have been caught if we ran tests on CentOS 7.

**Comment by andrewsykim** [comment]
Looks like we would need to use self-hosted runners for this https://github.com/actions/virtual-environments/issues/45

**Comment by cpuguy83** [comment]
Yes, for any specific targets we'd need self-hosted runners, otherwise we'll get Azure kernels.
